.chloggen/elasticsearchexporter_revert-array-dimension-workaround.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: elasticsearchexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Revert TSDB array dimension workaround for metrics OTel mode
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [35291]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: Remove the workaround to stringify array dimensions as the limitation has been lifted in Elasticsearch.
```

```suggestion
subtext: Remove the workaround to stringify array dimensions as the limitation has been lifted in Elasticsearch v8.16.0.
```

nit: I think adding a version would be helpful to the users. Also, should we mention that this change means that metrics OTel mode won't work well with <8.16.0 versions of ES?

---

I don't expect OTel mode to work well with <8.16 from the get go, not only related to this change. Maybe we should clarify it in README in another PR.

---

Sounds good!

---

+1 to adding docs that describe that the otel mode requires ES 8.16+